src/multiversioning.zig

```diff
@@ -1603,6 +1619,13 @@ pub fn print_information(
     try output.print("multiversioning.absolute_exe_path={s}\n", .{absolute_exe_path});
 
     const header = multiversion.target_header.?;
+
+    // `source_buffer` contains the same data as `target_file` - this code doesn't update anything
+    // after the initial open_sync().
+    try header.past.verify_checksums(
+        multiversion.source_buffer[multiversion.target_body_offset.?..],
```

Comment:
In the elf/etc parser functions should we also return body size to use? (I think we already parse it out.)